frontend/packages/dev-console/src/components/projects/ProjectListPage.tsx

```diff
@@ -4,12 +4,13 @@ import { ProjectsTable } from '@console/internal/components/namespace';
 import { PageHeading } from '@console/internal/components/utils';
 import './ProjectListPage.scss';
 
-export interface ProjectListPageProps {
+export type LazySubTitleRender = (props: () => void) => React.ReactNode;
+export type ProjectListPageProps = {
   title: string;
   listComponent?: React.ComponentType<any>;
-  children?: React.ReactNode;
+  children?: React.ReactNode | LazySubTitleRender;
```

Comment:
`ProjectListPage` children should still be `React.ReactNode` 
It's the props of `CreateProjectListPage` that need to change.

Reply:
yeah correct, did the mistake here. Have updated